src/lib.rs

```diff
@@ -21,9 +21,9 @@ struct Bundle {
 #[pymethods]
 impl Bundle {
     #[new]
-    fn new(namespace: &str, ftl_filenames: &'_ Bound<'_, PyList>) -> PyResult<Self> {
-        let langid_en: LanguageIdentifier = namespace.parse().expect("Parsing failed");
-        let mut bundle = FluentBundle::new_concurrent(vec![langid_en]);
+    fn new(language: &str, ftl_filenames: &'_ Bound<'_, PyList>) -> PyResult<Self> {
```

Comment:
I think this should be locale (at least, that is what the docs use: https://docs.rs/fluent/latest/fluent/bundle/struct.FluentBundle.html#method.new).

Reply:
Hmm... it's a bit confusing.

The reason I called it language here is because I think we are expecting it to be a [Unicode Language Identifier](https://unicode.org/reports/tr35/tr35.html#Unicode_language_identifier) rather than a locale, but I do take your point that it isn't consistent with the function name. But have I got it wrong there? It feels weird to me to call it locale if it is meant to be a language identfier, if that is actually the case.

Merging for now but happy to continue to iterate...